test/quantization/test_quantize_fx.py

```diff
@@ -717,26 +714,14 @@ def forward(self, x):
         original_ref_m.conv2.weight = torch.nn.Parameter(original_m.custom.conv.weight.detach())
         original_ref_m.conv2.bias = torch.nn.Parameter(original_m.custom.conv.bias.detach())
 
-        from torch.fx.symbolic_trace import Tracer
-
-        # define a custom tracer to not trace through the custom module
-
-        class CustomTracer(Tracer):
-            def is_leaf_module(self, m, module_qualified_name):
-                return (m.__module__.startswith('torch.nn') and
-                        not isinstance(m, torch.nn.Sequential)) or \
-                    isinstance(m, CustomModule)
-
         # TODO: add other quant types after mixed mode support
         for quant_type in [QuantType.STATIC]:
-            # register observed and quantized custom module classes
-            register_observed_custom_module_mapping(CustomModule, ObservedCustomModule)
-            register_quantized_custom_module_mapping(CustomModule, QuantizedCustomModule)
-
-            m = torch.fx.GraphModule(original_m, CustomTracer().trace(original_m))
-            qconfig_dict = {'': default_qconfig}
+            qconfig_dict = {
+                '': default_qconfig,
+                'custom_module_class':
```

Comment:
Do we want custom and standalone modules to be part of the qconfig dict?  Isnt this making the qconfig dict more complex? Can we handle CustomModule by defining the module mappings separately? Also, can standalone modules be a separate argument? This way qconfigDict only specifies qconfigs, not module mappings or other info.

Reply:
we can discuss, but I think it's easier for users to specify everything in qconfig_dict so that they don't need to remember other apis. these are optional arguments as well, only people who need this feature need to know about it.

Reply:
seems like currently (before this PR):
* key of qconfig dict defines the subgraph the rule applies to
* value of qconfig dict defines the observers used for the subgraph matching the key

I think it could be reasonable to make qconfig hold more information other than the observers, my question would be if we can make sure each type of information is only held in one place.  For example, if we are adding custom mappings to qconfig_dict, it would be ideal to make qconfig_dict the only place where custom mappings could be defined.  @jerryzh168 , are there any other ways to customize things right now other than the qconfig_dict?

Reply:
currently for custom modules user can call `register_observed_custom_module_mapping`, but we can make that api private